channels/applicationinsights-channel-js/Tests/Sender.tests.ts

```diff
@@ -60,7 +60,7 @@ export class SenderTests extends TestClass {
                     baseData: {}
                 };
                 try {
-                    this._sender.processTelemetry(telemetryItem);
+                    this._sender.processTelemetry(telemetryItem, null);
```

Comment:
The context is optional as existing plugin implementation may not pass it, so null is a valid value.
And it's really only required once we have a plugin that uses this value (i.e. a Singleton/shared plugin implementation)

Reply:
is including the `, null` necessary?